examples/example_seed_demonstration.py

```diff
@@ -31,17 +22,20 @@
 print('='*70)
 o.seed_elements(lon=4, lat=60, time=time)
 #o.run(steps=1)
+o.mode = Mode.Result
 o.plot(buffer=.7, fast=True)
 
 
 #%%
 # Seeding 100 elements within a radius of 1000 m
+o = OceanDrift(loglevel=50)
 print('\n' + '='*70)
 print('Seeding 100 elements within a radius of 1000 m:')
 print('o.seed_elements(lon=4, lat=60, number=100, radius=1000, time=time)')
 print('='*70)
 o.seed_elements(lon=4, lat=60, number=100, radius=1000, time=time)
 #o.run(steps=1)
+o.mode = Mode.Result
```

Comment:
Better to remove the `@require_mode` decorator on `def plot` than setting the mode.

Reply:
Yes, I now removed the requirement on `plot` and `animation` as these are made to also visualize seeding prior to the run